flink-runtime-web/src/test/java/org/apache/flink/runtime/webmonitor/WebFrontendITCase.java

```diff
@@ -217,12 +231,12 @@ public void getTaskManagerLogAndStdoutFiles() {
 	@Test
 	public void getConfiguration() {
 		try {
-			String config = TestBaseUtils.getFromHTTP("http://localhost:" + port + "/jobmanager/config");
+			String config = TestBaseUtils.getFromHTTP("http://localhost:" + CLUSTER.getWebUIPort() + "/jobmanager/config");
 
 			Map<String, String> conf = WebMonitorUtils.fromKeyValueJsonArray(config);
 			assertEquals(
-				cluster.configuration().getString("taskmanager.numberOfTaskSlots", null),
-				conf.get(ConfigConstants.TASK_MANAGER_NUM_TASK_SLOTS));
+				CLUSTER_CONFIGURATION.getString(ConfigConstants.LOCAL_START_WEBSERVER, null),
```

Comment:
what was up here? 😅

Reply:
I wanted an option for which the configured value is different from the default. The default for `numberOfTaskSlots` is one, so the test would've passed even if the handler returned a vanilla configuration.

Reply:
Yeah, that's what I meant, the previous code was a bit strange